integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/SimpleQueryStringSearchPredicateIT.java

```
@@ -231,6 +235,92 @@ public void noTokenAfterAnalysis() {
+
+		String whitespaceAnalyzedField = indexMapping.whitespaceAnalyzedField.relativeFieldName;
+		String whitespaceLowercaseAnalyzedField = indexMapping.whitespaceLowercaseAnalyzedField.relativeFieldName;
+
+		IndexSearchQuery<DocumentReference> query = scope.query()
+				.asReference()
+				.predicate( f -> f.simpleQueryString().onField( whitespaceAnalyzedField ).matching( "HERE | PANDA" ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_2 );
+
+		query = scope.query()
+				.asReference()
+				.predicate( f -> f.simpleQueryString().onField( whitespaceLowercaseAnalyzedField ).matching( "HERE | PANDA" ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_1, DOCUMENT_2, DOCUMENT_3 );
+
+		query = scope.query()
+				.asReference()
+				.predicate( f -> f.simpleQueryString().onField( whitespaceAnalyzedField ).matching( "HERE | PANDA" )
+						.analyzer( OverrideAnalysisDefinitions.ANALYZER_WHITESPACE_LOWERCASE.name ) )
+				.toQuery();
+
+		assertThat( query )
+				.hasDocRefHitsAnyOrder( INDEX_NAME, DOCUMENT_1 );
+	}
+
+	@Test
+	public void analyzerOverride_notExistingName() {
+		StubMappingSearchScope scope = indexManager.createSearchScope();
+		String whitespaceAnalyzedField = indexMapping.whitespaceAnalyzedField.relativeFieldName;
+
+		SubTest.expectException( () -> scope.query()
+				.asReference()
+				.predicate( f -> f.simpleQueryString().onField( whitespaceAnalyzedField ).matching( "HERE | PANDA" )
+						// we don't have any analyzer with that name
+						.analyzer( "this_name_does_actually_not_exist" ) )
+				.toQuery().fetch()
+		)
+				.assertThrown()
+				.isInstanceOf( SearchException.class )
+
+		// TODO on Elasticsearch backend the error is not handled by Search
+		//.hasMessageContaining( "HSEARCH000353: Unknown analyzer: 'this_name_does_actually_not_exist'. Make sure you defined this analyzer." )
+		;
+	}
+
+	@Test
+	public void analyzerIgnore() {
```

Same comment about skipping analysis for normalized fields: currently the option is silently ignored for normalized fields, that's probably not a good idea.